aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml

```diff
@@ -136,7 +136,8 @@ Resources:
             sudo systemctl start amazon-ssm-agent
 
             #setup traffic mirroring network interface
-            sudo ip link add capture0 type vxlan id 4058602 local 10.0.0.83 remote 10.0.0.84 dev eth1 dstport 4789
+            eth=$(ls /sys/class/net | grep eth)
```

Comment:
1. you can have more than 1 interface, add `| sort | head -n1`
2. i assume you want `eth*` regex and not `*eth*`, use `grep -E "^eth*"`

Reply:
consider using `route | grep default | awk '{print $8}'` to get the default interface name